SDKGenerator.njsproj

```diff
@@ -754,6 +754,15 @@
     <Content Include="targets\XPlatCppSdk\source\cppsdk\configLinux.json" />
     <Content Include="targets\XPlatCppSdk\source\cppsdk\configWindows.json" />
     <Content Include="targets\XPlatCppSdk\source\cppsdk\include\playfab\PlayFabBaseModel.h" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\LibUnitTest.vcxproj" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\LibUnitTest.vcxproj.filters" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\packages.config" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\PlayFabClientTest.cpp" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\PlayFabEntityTest.cpp" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\PlayFabServerTest.cpp" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\stdafx.cpp" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\stdafx.h" />
+    <Content Include="targets\XPlatCppSdk\source\LibUnitTest\targetver.h" />
```

Comment:
Were those just forgotten to be included into project? (since the files themselves are missing in PR)